components/IndicatorStat/utils/data.ts

```diff
@@ -12,21 +12,21 @@ const getLocalValue = (data: LocationData, options: ValueOptions): string => {
   if (data.meta) {
     const meta: LocationDataMeta = JSON.parse(data.meta);
     if (meta.valueLocalCurrency) {
-      return addPrefixAndSuffix(formatNumber(meta.valueLocalCurrency), options);
+      return addPrefixAndSuffix(formatNumber(meta.valueLocalCurrency, 1), options);
```

Comment:
This default is configured in the `formatNumber` function... not needed here.

Reply:
updated